lib/uwb/UwbSession.cxx

```diff
@@ -47,12 +47,10 @@ UwbSession::StartRanging()
 {
     PLOG_VERBOSE << "start ranging";
     bool rangingActiveExpected = false;
-    const bool wasRangingActive = m_rangingActive.compare_exchange_weak(rangingActiveExpected, true);
-    if (wasRangingActive) {
-        return;
+    const bool wasNotRangingActive = m_rangingActive.compare_exchange_weak(rangingActiveExpected, true);
+    if (wasNotRangingActive) {
```

Comment:
This feels a little awkward to say. How about:
```suggestion
    const bool wasRangingInactive = m_rangingActive.compare_exchange_weak(rangingActiveExpected, true);
    if (wasRangingInactive) {
```

Reply:
Yeah that's much better, thanks!